libbeat/docs/outputconfig.asciidoc

```diff
@@ -198,7 +198,7 @@ operation doesn't succeed after
 ===== bulk_max_size
 
 The maximum number of events to bulk in a single Elasticsearch bulk API index request.
-The default is 50.
+The default is 200.
```

Comment:
Ah, there's a confusion here. We've changed bulk_max_size' default for the Logstash output, where this setting is not documented at all (no idea why). Might be worth for me or @urso (hint) to come up with a longer explanation since this is critical for performance.